tests/CPMmodel/graphModel.py

```diff
@@ -0,0 +1,26 @@
+from LOGOS.src.CPM.PertMain2 import Pert
+from LOGOS.src.CPM.PertMain2 import Activity
+
+class project():
+  start = Activity("start", 10)
+  b     = Activity("b",     20)
+  c     = Activity("c",      5)
+  d     = Activity("d",     10)
+  f     = Activity("f",     15)
+  g     = Activity("g",      5)
+  h     = Activity("h",     15)
+  end   = Activity("end",   20)
+
+  activitiesList = [start,b,c,d,f,g,h,end]
+
+  graph = {start: [f,b,h], 
+           b    : [c], 
+           c    : [g,d], 
+           d    : [end], 
+           f    : [g], 
+           g    : [end],
+           h    : [end],
+           end  : []}
+
```

Comment:
Do the users need to create the graphModel? or they only need to provide the inputs through input xml, and we can automatically create the graph model?